package.json

```diff
@@ -42,7 +42,8 @@
         "preinteg": "npm run build:dev",
         "integ": "testcafe --app \"http-server ./build/dev -s\"",
         "integ:browser": "npm run integ -- --config-file=./.testcafe.browser.js",
-        "integ:headless": "npm run integ -- --config-file=./.testcafe.headless.js",
+        "integ:headless-linux": "npm run integ -- --config-file=./.testcafe.headless-linux.js",
+        "integ:headless-windows": "npm run integ -- --config-file=./.testcafe.headless-windows.js",
```

Comment:
nit: We can update our internal runbooks regarding the changes in how we can run the integ tests.